.github/workflows/ci.yml

```diff
@@ -111,6 +111,9 @@ jobs:
     - run: yarn diff:sample-json
       if: matrix.os == 'windows-latest'
 
+    # Fail if any changes were written to source files (ex, from -GA).
+    - run: git diff --exit-code
```

Comment:
Wanna add to the smokes step too?

Reply:
yeah I'll add everywhere I just noticed it happens on update:sample-json too